pavelib/utils/envs.py

```diff
@@ -82,118 +70,41 @@ class Env:
     # build steps. For local development/testing, this shouldn't be needed.
     if os.environ.get("SHARD", None):
         shard_str = "shard_{}".format(os.environ.get("SHARD"))
-        BOK_CHOY_REPORT_DIR = BOK_CHOY_REPORT_DIR / shard_str
-        BOK_CHOY_LOG_DIR = BOK_CHOY_LOG_DIR / shard_str
-
-    # The stubs package is currently located in the Django app called "terrain"
-    # from when they were used by both the bok-choy and lettuce (deprecated) acceptance tests
-    BOK_CHOY_STUB_DIR = REPO_ROOT / "common" / "djangoapps" / "terrain"
 
     # Directory that videos are served from
     VIDEO_SOURCE_DIR = REPO_ROOT / "test_root" / "data" / "video"
 
-    PRINT_SETTINGS_LOG_FILE = BOK_CHOY_LOG_DIR / "print_settings.log"
+    PRINT_SETTINGS_LOG_FILE = GEN_LOG_DIR / "print_settings.log"
 
     # Detect if in a Docker container, and if so which one
-    SERVER_HOST = os.environ.get('BOK_CHOY_HOSTNAME', '0.0.0.0')
+    SERVER_HOST = os.environ.get('0.0.0.0')
```

Comment:
```suggestion
    SERVER_HOST = '0.0.0.0'
```

What is `SERVER_HOST` being used for? Can we get rid of this setting?